src/storage/mvcc/reader/mod.rs

```diff
@@ -113,7 +113,7 @@ impl<S: Snapshot> MvccReader<S> {
 
     pub fn load_lock(&mut self, key: &Key) -> Result<Option<Lock>> {
         if self.scan_mode.is_some() && self.lock_cursor.is_none() {
-            let iter_opt = IterOption::new(None, None, true);
+            let iter_opt = IterOption::new(None, None, self.fill_cache);
```

Comment:
@zhangjinpeng1987 here

Reply:
I think it is better to fill cache for lock cf. @breeswish Do you have any consideration for this which i don't notice?

Reply:
It was for benchmarks that operate on fill_cache=false for more stable results. @zhangjinpeng1987 Do you think this is necessary? I can discard this change anyway.